client/lib/application/src/application-list/application-list.hbs

```diff
@@ -3,14 +3,48 @@
     <a href="/application/create" class="btn btn-default">
     <i class="fa fa-plus"></i> Create Application</a>
 </div>
-{{#if apps.length}}
-    <ul>
-        {{#each apps}}
+<div class="form-group">
+    <form class="form">
+        <label for="yourturn-search">Search:</label>
+        <div class="input-group">
+            <input
+                name="yourturn-search"
+                autofocus="autofocus"
+                value="{{term}}"
+                type="search"
+                aria-label="Enter your term"
+                data-action="search"
+                placeholder="Application" />
+            <button
+                type="submit"
+                class="btn btn-default">
+                    <i class="fa fa-search"></i> Search
+                </button>
+            </div>
+    </form>
+</div>
+<h3>Your team apps</h3>
```

Comment:
So.

* [UX StackExchange](http://ux.stackexchange.com/questions/24612/my-account-or-your-account) goes with "Your".
* [Microsoft UX Guidelines](https://msdn.microsoft.com/en-us/library/dn742477.aspx) is like "it depends", but would use "Your" here.
* [dcurtis](http://dcurt.is/yours-vs-mine) also goes with "Your".

So I’d call it "Your Applications". What do you think?